packages/android-client/FishjamClient/src/main/java/fishjam/media_events/CandidateKt.kt

```diff
@@ -0,0 +1,106 @@
+// Generated by the protocol buffer compiler. DO NOT EDIT!
```

Comment:
idea for future: would it be possible to keep this generated files in folder called `generated`?

Reply:
Unfortunately it looks like it's not possible, as the package name comes from the protofile and `protoc` does not provide the ability to modify it. So although we could change the file location it won't work on Android as it expects certain folder structure. 

We could modify the protos file, but this might destroy code generation for others...